src/test/scenarios/managed-network/output/src/managed-network/azext_managed_network/generated/custom.py

```diff
@@ -267,7 +284,9 @@ def managed_network_managed_network_peering_policy_mesh_topology_update(instance
 def managed_network_managed_network_peering_policy_delete(cmd, client,
                                                           resource_group_name,
                                                           managed_network_name,
-                                                          policy_name):
-    return client.begin_delete(resource_group_name=resource_group_name,
-                               managed_network_name=managed_network_name,
-                               managed_network_peering_policy_name=policy_name)
+                                                          policy_name,
+                                                          no_wait=False):
+    return sdk_no_wait(no_wait, client.begin_delete,
+                                                   resource_group_name=resource_group_name,

```

Comment:
there may be some python formatting lint issue here?

Reply:
I am sure this one will not cause the lint error. but please don't merge this PR for now